projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -51,6 +59,10 @@ Running this query enables to detect potential issues in the class hierarchy and
 
 3. Detecting Unused Object Properties (level 7 - 2 points)
 
+Title: Unused Object Property Detection
+Constraint Description: Identifies object properties that are defined but not used in any instances.
+Severity: Warning
+
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```

Comment:
This one should work, I am not entirely sure how helpful it will be since CCO doesn't have many instances, but it's fine to leave it